tools/tsconfig.json

```diff
@@ -18,6 +18,7 @@
     "skipLibCheck": true,
     "skipDefaultLibCheck": true,
     "incremental": true,
+    "useUnknownInCatchVariables": false,
```

Comment:
How many errors are reported without this one turned off? 🤔

Reply:
@bbarthec https://github.com/expo/expo/runs/4084290325?check_suite_focus=true

Reply:
Ok, far too many to try and add everywhere explicit `catch (err: any)` 😅